04-tota11y-plugin.md

```diff
@@ -0,0 +1,18 @@
+ # 04. Use tota11y to Visualize Accessibility Issues
+
+**[📹 Video](https://egghead.io/lessons/aria-use-tota11y-to-visualize-accessibility-issues)**
+
+
+For [Chrome](https://chrome.google.com/webstore/detail/tota11y-plugin-from-khan/oedofneiplgibimfkccchnimiadcmhpe?hl=en)
+
+For [Firefox](https://addons.mozilla.org/en-US/firefox/addon/tota11y-accessibility-toolkit/)
+
+* You can toggle the extension in the extension menu of your chosen browser. Once you do that, you'll see the sunglasses icon on the bottom left of your screen.
```

Comment:
> you'll see the sunglasses icon on the bottom left of your screen.

An annotated screenshot of this could be worth the effort vs the text description